### ISSUE TITLE
ilife09 catalog

### ISSUE THREAD

**Issue by GoogleCodeExporter** [n8felton/instadmg#75]
```
checked out latest (408). Noticed someones local ilife DVD referenced in 
ilife09 update catalog. 

/Volumes/Vertexd is referenced

that line should probably get removed

```

Original issue reported on code.google.com by `w...@simplymac.ca` on 24 Nov 2010 at 4:15

**Comment by GoogleCodeExporter** [comment]
```
Thanks Wade! I've modified that to be more generic and commented out for an 
update I'll be posting today, good catch.  The only reason I'm not pushing it 
right now(along with Sarai 5.0.3 for 10.5) is I haven't tested for Digital 
Raw3.4 yet, could you please let me know if softwareupdate for your image asked 
for DigitalRaw3.4?  I know it didn't ask me for it last week, but we've only 
included 3.2, which is from April.  Thanks again, Allister 
```

Original comment by `allister...@gtempaccount.com` on 24 Nov 2010 at 11:48
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
As of r409, thanks again.
```

Original comment by `allister...@gtempaccount.com` on 25 Nov 2010 at 4:55
- Changed state: **Fixed**
